src/ASM/FiniteElement.h

```diff
@@ -114,6 +114,8 @@ class MxFiniteElement : public FiniteElement
 protected:
   //! \brief Writes the finite element object to the given output stream.
   virtual std::ostream& write(std::ostream& os) const;
+  //! \brief Global Output stream operator.
+  friend std::ostream& operator<<(std::ostream& os, const MxFiniteElement& fe);
```

Comment:
The conclusion on this was that it is _not_ needed, @akva2 probably triggered a bug in the compiler, or something. I just pushed the one-liner fix (line 45) plus a test-program, such that this commit can now be removed from this PR.